packages/api/utils/errorHandlingMiddleware.ts

```diff
@@ -0,0 +1,34 @@
+import middy from "@middy/core";
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { serializeError } from "serialize-error";
+import { ValidationErrorResponse } from "./response";
+import { REQUEST_BODY_INVALID } from "./errors";
+
+const errorHandlingMiddleware = (): middy.MiddlewareObj<APIGatewayProxyEvent, APIGatewayProxyResult> => {
+  const onError: middy.MiddlewareFn<APIGatewayProxyEvent, APIGatewayProxyResult> = async (
+    request
+  ): Promise<ValidationErrorResponse | void> => {
+    const error = serializeError(request.error)!;
+    // Catch middy validation error, wrap it in ValidationErrorResponse
+    if (error.message === "Event object failed validation") {
+      return new ValidationErrorResponse("Request failed validation", error.details as Record<string, unknown>);
+    }
+    if (error.message === "Business rules validation failed") {
+      const errorMap = error.details as Record<string, unknown>;
+      const businessRules = errorMap.input_validation_errors;
+      // should we keep this inside of input_validation_errors?
```

Comment:
Team: Thoughts on this comment?  `input_validation_errors` was a convention I made up.  In similar discussion with @juliusfitzhugh-ccpo I think I preferred to flatten this and remove `input_validation_errors`.

Reply:
`input_validation_errors` is currently removed, and the thumbs up to @jasonburkert-ccpo comment indicates that we should keep it that way.

Reply:
This appears to have been addressed (remove input_validation_errors) in `requestValidation.ts`